Userland/Libraries/LibGUI/TabWidget.cpp

```diff
@@ -370,6 +372,31 @@ void TabWidget::set_bar_visible(bool bar_visible)
 }
 
 Gfx::IntRect TabWidget::button_rect(size_t index) const
+{
+    if (this->has_vertical_tabs())
+        return vertical_button_rect(index);
+    else
+        return horizontal_button_rect(index);
```

Comment:
Nit-pick: When the `if` ends in a `return`, we prefer to skip the `else`:
```suggestion
    return horizontal_button_rect(index);
```